pkg/i2gw/providers/istio/converter_test.go

```diff
@@ -1245,6 +1245,96 @@ func Test_converter_convertVsHTTPRoutes(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "hosts with '*'",
+			args: args{
+				virtualService: metav1.ObjectMeta{
+					Name:      "test",
+					Namespace: "ns",
+				},
+				istioHTTPRoutes: []*istiov1beta1.HTTPRoute{
+					{
+						Headers: &istiov1beta1.Headers{
+							Request: &istiov1beta1.Headers_HeaderOperations{
```

Comment:
These fields are irrelevant for the test. Could you minimise the istio.HTTPRoute definition for the test case? (e.g. to keep only only header operation: request+add).